### PR TITLE
Added: the script, css, gulpfile and updated readme for bopis shopify(#1x63rug)

### PR DESCRIPTION
**Pull request by ymaheshwari1**

### CODE REVIEW

File: scripts/script-tag.js [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+        event.preventDefault();
+        event.stopImmediatePropagation();
+                
+        // let merchant define the behavior whenever pick up button is clicked, merchant can define an event listener for this event
+        jQueryBopis(document).trigger('prePickUp');
+
+        let facilityIdInput = jQueryBopis(`<input id="hc-pickupstore" name="properties[pickupstore]" value=${store.storeCode} type="hidden"/>`)
+        addToCartForm.append(facilityIdInput)
+
+        let facilityNameInput = jQueryBopis(`<input id="hc-pickupstore" name="properties[Pickup Store]" value="${store.locationName}, ${store.address.addressLines[0]}, ${store.address.locality}, ${store.address.administrativeArea}, ${store.address.postalCode}, ${store.address.regionCode}" type="hidden"/>`)
```
Comment (adityasharma7) [inline scripts/script-tag.js]: id should be unique

File: scripts/script-tag.js [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+
+            //check whether the storeCard contains any children, if not then displaying error
+            if (!jQueryBopis('.hc-store-information').children().length) {
+                jQueryBopis('.hc-store-not-found').html('No stores found for current product');
+            }
+        } else {
+            jQueryBopis('.hc-store-not-found').append('No stores found for current product');
+        }
+
+        jQueryBopis('h4:empty').hide();
```
Comment (adityasharma7) [inline scripts/script-tag.js]: Find in modal and then hide

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: scripts/script-tag.js (hunk 1 above)
Comment (ymaheshwari1) [reply scripts/script-tag.js]: Made the two ids unique

File: scripts/script-tag.js (hunk 2 above)
Comment (ymaheshwari1) [reply scripts/script-tag.js]: Sir, Improved code to use find method in the current modal

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**